az-digital/az_quickstart
# [WIP] Closes #2669: Update search.page.content_exclude_az_content.yml

## Description
When I search the site, I don't want old events to come up before upcoming events.

## Related issues




Closes #2669

## How to test



1. Create new and old events with a common phrase
2. Search for the common phrase
3. New/upcoming events should come up first in the list

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
